routes/web.php

```diff
@@ -241,7 +241,17 @@
     });
 
     if (Features::isEnabled('hxl')) {
-        $router->get('hxl', "HXLController@index");
+		$router->group([
+			'namespace' => 'HXL',
+			'prefix' => 'hxl',
+			'middleware' => ['auth:api'] //TODO which scopes do we need?
+		], function () use ($router) {
+			// Public access
+			$router->get('/', 'HXLController@index');
+			$router->get('/licenses', 'HXLLicensesController@index');
+			$router->get('/tags', 'HXLTagsController@index');
+			$router->get('/form_attributes', 'HXLTagAttributesController@index');
```

Comment:
I hate this endpoint's name. It's in the spec but it's awful. I don't know what to name it yet

Reply:
form_attribute_types maybe? I don't know